docs/en/tutorials/how_to_install_mmdeploy_on_jetsons.md

```diff
@@ -20,9 +27,11 @@ There are two major installation methods including,
 1. SD Card Image Method
 2. NVIDIA SDK Manager Method
 
-You can find a very detailed installation guide from NVIDIA [official website](https://developer.nvidia.com/jetpack-sdk-50dp).
+You can find a very detailed installation guide from NVIDIA [official website](https://developer.nvidia.com/jetpack-sdk-461).
+
+**Note:** Please select the option to install "Jetson SDK Components" when using NVIDIA SDK Manager as this includes CUDA and TensorRT which are needed for this guide.
```

Comment:
We recommend the following note style because it can be rendered prettier on the readthedocs.
```{note}
```

Reply:
It is fixed